### PR TITLE
Feature 42 scripts dir

### LINKED ISSUE #42: Add new 'scripts' directory and create includes for PBS script in docs
## Describe the Task ##
Add new 'scripts' directory and create includes for PBS script in docs

### Time Estimate ###
<1 day

### Relevant Deadlines ###
2.0 release

### New Parameters ###
None

### Technical References ###
None

### Assignee ###
- [x] Assign this Task to the appropriate person

### Labels ###
- [x] Select **pillar**
- [x] Select **priority**
- [x] Select **requestor(s)**

### PR DESCRIPTION
**Pull request by jprestop**
## Expected Differences ##

- [x] Do these changes introduce new tools, command line arguments, or configuration file options? **[Yes]**</br>
If **yes**, please describe:</br>
They include two batch scripts for running on Casper and Derecho.

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>
Added new directories and new scripts

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>
Review the added directories, scripts, and changes to the documentation (can view here: https://fasteddy-model.readthedocs.io/en/feature_42_scripts_dir/run_ncar_hpcs.html)
Note: I added a post-processing directory, but it looks like GitHub is ignoring it until there is content in it.  We can either wait to merge this until I can add the binary to NetCDF converter script for post-processing from @jsauer-NCAR or we can add it later, whatever you prefer.

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the test suite? **[N/A]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **[At your convenience, before the release]**.</br>

## Pull Request Checklist ##
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Select: **Reviewer(s)**
- [x] After submitting the PR, select the :gear: icon in the **Development** section of the right hand sidebar. Search for the issue that this PR will close and select it, if it is not already selected.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

**Review by domingom — COMMENTED** [review]
We perhaps want to have a parent directory named 'scripts' and then inside of it have 'batch_jobs' and others like 'post-processing'. It seems like right now there is just one folder with the name 'scripts/batch_jobs'.

**Comment by jprestop** [comment]
@domingom It only looks that way because there is only one directory in there now.  Once we have the script for the `python_utiltiies/post-processing` subdirectory, the "batch_jobs" part will no longer show up and it'll be left with just "scripts".  I've seen this behavior in our other repos. 
@domingom and @jsauer-NCAR, why don't we hold off on approving this PR until I get the post-processing script from Jeremy and can add it in here?  Does that sound good?

**Comment by domingom** [comment]
Thanks for the clarification @jprestop. Sounds good to me to wait for the post-processing script before merging this PR.

**Review by jsauer-NCAR — DISMISSED** [review]

**Review by jsauer-NCAR — APPROVED** [review]

**Pull request merged**
